src/physics/ra_driver.f90

```diff
@@ -239,7 +239,13 @@ subroutine rad(domain, options, dt, halo, subset)
         qs = 0
 
         cldfra=0
-        albedo=0.17
+
+        if (options%lsm_options%monthly_albedo) then
+            ALBEDO = domain%albedo%data_3d(:, domain%model_time%month, :)
+        else
+            ALBEDO = domain%albedo%data_3d(:, 1, :)
```

Comment:
The CI is failing here since with our test case `domain%albedo%data_3d` hasn't been allocated.

Maybe we add a check for if it hasn't been allocated
```
if else (.not. associated(domain%albedo%data_3d)) then
    ALBEDO = 0.17
else
     ....
```

Or do something else?